hotModuleReplacement.js

```diff
@@ -0,0 +1,18 @@
+module.exports = function(compilationHash, publicPath, outputFilename) {
+  if (document) {
+    var styleSheets = document.getElementsByTagName('link');
+    for (var i = 0; i < styleSheets.length; i++) {
+      if (styleSheets[i].href) {
+        var hrefUrl = styleSheets[i].href.split('?');
+        var href = hrefUrl[0];
+        var hash = hrefUrl[1];
+        if (hash !== compilationHash && href === document.location.origin + publicPath + outputFilename) {
```

Comment:
This will definitely break with different-origin public paths, but then again hot reloading overall wouldn't work in that case either.

`location.origin` is also not available in IE.